src/lib/containers/entity/page/CreatedByModifiedBy.tsx

```diff
@@ -0,0 +1,170 @@
+
+  const { data: tableQueryResult } = useGetQueryResultBundleWithAsyncStatus(
+    {
+      entityId,
+      query: {
+        sql: `SELECT * FROM ${entityIdWithVersion} LIMIT 0`,
+      },
+      partMask: BUNDLE_MASK_LAST_UPDATED_ON,
+      concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    },
+    { enabled: !!(entity && isTable(entity)) },
+  )
+
+  const tableLastRebuilt = tableQueryResult?.responseBody?.lastUpdatedOn
+  const friendlyName = entity
+    ? entityTypeToFriendlyName(convertToEntityType(entity.concreteType))
+    : ''
+
+  const datasetCreatedByTooltipText =
+    'This is the user who created this Dataset. ' +
+    'This may not be the same person who generated the files in this Dataset, or who originally uploaded these files to Synapse.'
+
+  const tableModifiedOnTooltipText = (
+    <>
+      <p>
+        This is when the configuration of this {friendlyName} was last changed.
+      </p>
+      <p>Configuration changes may be triggered by (for example):</p>
+      <ul>
+        <li>Editing the name of the {friendlyName}</li>
+        <li>Updating the schema used by the {friendlyName}</li>
+      </ul>
+      <p>
+        This does NOT reflect changes to the data displayed in the the{' '}
+        {friendlyName}.
+      </p>
+    </>
+  )
+  /*
+   If the tooltip text is a string, MUI automatically applies it to aria-label which is more accessible and testable.
+    Since this tooltip text is a ReactNode, let's manually create a short string to use for the aria-label.
+   */
+  const tableModifiedOnAccessibleLabel = `This is when the configuration of this ${friendlyName} was last changed.`
+
+  const tableLastUpdatedTooltipText =
+    `When data changes, the ${friendlyName} is rebuilt to reflect those changes. ` +
+    `This is the last time changes were made to the data.`
+
+  const createdByTooltipId = `${entityIdWithVersion}-createdByTooltip`
+  const modifiedByTooltipId = `${entityIdWithVersion}-modifiedByTooltip`
+  const lastUpdatedTooltipId = `${entityIdWithVersion}-lastUpdatedTooltip`
+
+  const iconSx: SxProps = {
+    width: '16px',
+    height: '16px',
+    ml: '4px',
+    verticalAlign: 'text-bottom',
+  }
+
+  return (
+    <Box sx={{ bgcolor: 'grey.100', px: '40px', py: '10px' }}>
+      <Breadcrumbs separator={<Separator />}>
+        <ConditionalWrapper condition={!entity} wrapper={Skeleton}>
+          <Typography
+            sx={{ color: 'grey.700' }}
+            variant={'breadcrumb1'}
+            aria-describedby={createdByTooltipId}
+          >
+            {friendlyName} created by{' '}
+            <UserCard ownerId={entity?.createdBy} size={SMALL_USER_CARD} /> on{' '}
+            {formatDate(dayjs(entity?.createdOn))}
+            {entity && isDataset(entity) && (
+              <Tooltip
+                id={createdByTooltipId}
+                title={datasetCreatedByTooltipText}
+              >
+                <InfoTwoTone sx={iconSx} />
+              </Tooltip>
```

Comment:
Directly pulling the MUI icon because I needed more control over the tooltip than IconSvg gives me...if preferable I can refactor IconSvg instead.